packages/adp-tooling/src/preview/change-handler.ts

```diff
@@ -7,6 +7,7 @@ import { render } from 'ejs';
 import { randomBytes } from 'crypto';
 
 const OBJECT_PAGE_CUSTOM_SECTION = 'OBJECT_PAGE_CUSTOM_SECTION';
+const ADD_HEADER_FIELD = 'ADD_HEADER_FIELD';
```

Comment:
Template name should match the convention - prefix with the page (if applicable) and it shouldn't contain verbs.
```suggestion
const OBJECT_PAGE_HEADER_FIELD = 'OBJECT_PAGE_HEADER_FIELD';
```